src/dune_rules/lib_info.mli

```diff
@@ -86,9 +86,25 @@ module Main_module_name : sig
   type t = Module_name.t option Inherited.t
 end
 
+(** What's the subset of fields that uniquely identifies this stanza? *)
```

Comment:
Could you move this to its own file? Also `Lib_id` sounds just as good and shorter to me.